src/frontends/ir/tests/frontend_test.cpp

```diff
@@ -0,0 +1,1366 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Please use copyright as we have in other headers
```suggestion
Copyright (C) 2018-2022 Intel Corporation
```